bazel/repository_locations.bzl

```diff
@@ -258,8 +258,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Luajit2",
         project_desc = "Openresty/luajit2 - OpenResty's maintained branch of LuaJIT",
         project_url = "https://github.com/openresty/luajit2",
-        version = "5db6cb9f3ad28efa77fa6807dbe7a610cd39bd55",
-        sha256 = "2bd0dab65206f05f6c60c144a84124a12b7fa292bc8cfebf66f4d00cbded62ef",
+        version = "1085a4d562b449e7be9e4508b52a19651bdf04a6",
```

Comment:
Do we need to the release-date field as well?

Reply:
Done